src/javaRosa/plugin.js

```diff
@@ -567,9 +567,26 @@ define([
             function itextValidator(property, name) {
                 return function (mug) {
                     var itext = mug.p[property],
-                        hasItext = itext && itext.hasHumanReadableItext();
+                        hasItext = false,
+                        languages = itext.itextModel.languages;
+                    if (itext) {
+                        _.each(['default', 'long', 'short'], function(form) {
+                            if (itext.hasForm(form)) {
+                                hasItext = true;
+                                if (_.find(languages, function(lang) {
+                                    return !itext.get('default', lang);
+                                })) {
+                                    hasItext = false;
+                                }
```

Comment:
Maybe it would make sense to add some unit tests for `ItextItem.hasHumanReadableItext` for various permutations of itext items with and without "human readable itext"?